README.md

```diff
@@ -323,7 +323,7 @@ Write code on a whiteboard, not a computer. Test with some sample inputs. Then t
     - [ ] [Machine Code Instructions (video)](https://www.youtube.com/watch?v=Mv2XQgpbTNE)
 
 - [ ] **How floating point numbers are stored:**
-    - [ ] simple 8-bit: [Fractions in binary? (video)](http://math.stackexchange.com/questions/301435/fractions-in-binary)
+    - [ ] simple 8-bit: [Fractions in binary? (video)] (There is an error in calculation in video - see description) (http://math.stackexchange.com/questions/301435/fractions-in-binary)
```

Comment:
Hi Artur, please move the annotation inside the [] portion. This is a markdown-style link and would break it:

format:
[link text](link url)